### PR TITLE
Add shell script to auto increment version number

### PR DESCRIPTION
**Pull request by Filbird**
This script will increment the patch number with every Wercker build.
VERSION file gets updated with new value.

### CODE REVIEW

**Comment by stjohnjohnson** [comment]
Shouldn't you be using tags?  If not, you need to check this file in each time.  @jer

**Comment by jer** [comment]
Yep, good start but we need to at least get all git tags and sort them numerically. Then we need to bump the largest version, tag the repo, push the tags.

File: wercker.yml [hunk 1]
```diff
@@ -13,11 +13,17 @@ build:
 deploy:
   steps:
     - script:
-        name: get version from VERSION file
-        code: export VERSION=$(cat VERSION)
+        name: get highest version from tags
+        code: export VERSION=$(sh incrementVersion.sh)
+    - script:
+        name: add new annotated tag
+        code: git tag -a $VERSION -m "Auto-generated tag message"
```
Comment (Filbird) [inline wercker.yml]: What annotated message should the tag have?

**Comment by Filbird** [comment]
Shell script now gets the version from the highest git tag value rather than a local file.
Added steps in the wercker.yml to generate a new tag and push it.

File: wercker.yml (hunk 1 above)
Comment (jer) [inline wercker.yml]: How about "Tagging release version x.y.z"?

Comment (Filbird) [inline wercker.yml]: sure! Will change

**Comment by jer** [comment]
👍

**Pull request merged**